toolkit/stable_diffusion_model.py

```diff
@@ -1065,6 +1076,13 @@ def generate_images(
                     gen_config = image_configs[i]
 
                     extra = {}
+
+                    if self.controlnet:
+                        extra['control_image'] = gen_config.control_image
```

Comment:
why are you takking this from config? this should match the specific sample you ar generating.. or are you always generating with the same condition image when sampling?

Reply:
I don't have an edge map for an image I didn't generate yet, I think the generic one is fine.